docs/source/sysadmin_guide/log_configuration.rst

```diff
@@ -0,0 +1,139 @@
+*****************
+Log Configuration
+*****************
+
+Overview
+========
+The validator and the Python SDK make it easy to customize the logging output.
+This is done by creating a TOML (`<https://github.com/toml-lang/toml>`_)
+formatted log config file and passing it to the built-in Python logging module.
+
+Default
+=======
+
+If there is no log configuration file provided, the default is to create an
+error log and a debug log. Theses files will be stored in the log directory that
+is configured by the SAWTOOTH_HOME environment variable. For example, if
+`SAWTOOTH_HOME="/home/ubuntu/sawtooth"`, the logs will be written
+to `/home/ubuntu/sawtooth/logs`.
+
+If SAWTOOTH_HOME is not set, the default location is OS-specific. On Linux,
+it's `/var/log/sawtooth`. On Windows, it is one level up from the where the
+command is installed. For example, if `C:\\sawtooth\\bin\\validator.exe`
+starts the validator, the log directory is `C:\\sawtooth\\logs\\`. The log
+directory can also be set in the path.toml file, however, this file should
+rarely be used.
+
+The name of the validator logs will look like the following:
+
+- validator-{first 8 characters of the validator's pubkey}-debug.log
+- validator-{first 8 characters of the validator's pubkey}-error.log
+
+Examples:
+
+- *validator-03aa0272-debug.log*
+- *validator-03aa0272-error.log*
+
+For Python transaction processors, the author determines the name of the log
+file. It is highly encouraged that the file names are unique for each running
+processor to avoid naming conflicts.  The example transaction processors
+provided with the SDK uses the following:
+
+- {tp_name}-{zmq_identity}-debug.log
+- {tp_name}-{zmq_identity}-error.log
+
+Examples:
+
+-  *intkey-18670799cbbe4367-debug.log*
+-  *intkey-18670799cbbe4367-error.log*
+
+Configuration
+=============
+
+To change the default behavior, a log configuration file can be placed in the
+config directory.
```

Comment:
A `(see below)` would be nice here. On first read, I thought I had missed where the config directory was defined.